### DISCUSSION TITLE
petraptor inside submarine eats whole abyss monster trough hull

### DISCUSSION THREAD

**Discussion by DrBruhman** [FakeFishGames/Barotrauma#13623] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Raptor sever some segments of dead doomworm that slightly pokes bottom of submarine and can be seen from inside.

### Reproduction steps

Submarine with hungry petraptor inside + corpse of abyss creature

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### -

May work in any other.?

### Version

Other

### -

1.1.18 or later i guess 

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
nah
```

**Comment by Regalis11** [comment]
Thank you for the report!

I unfortunately could not find a way to reproduce the issue. I couldn't get the pet to eat a character that's outside the submarine. Since doomworms don't appear in the vanilla game, I assume you're using some kind of mods? Are you e.g. using something like the [Let Them All](https://steamcommunity.com/sharedfiles/filedetails/?id=2805065898) In which could potentially cause these kinds of issues by allowing the large monsters to be technically inside the sub (even though they don't actually fit inside)?

**Comment by DrBruhman** [comment]
I was not used any server-side mods while i encountered this bug, also everyone on server(only my friend) have seen it.

**Comment by DrBruhman** [comment]
we was been on modified humpback, so maybe i should try to replicate it with this sub
